pan3d/ui/coordinate_configure.py

```diff
@@ -42,7 +47,7 @@ def __init__(
                                     "%s?.size - 1" % coordinate_info,
                                     0,
                                 ),
-                                step=1,
+                                step=("%s?.step" % coordinate_info, 0),
```

Comment:
f string?

Reply:
Changed in [28561cb](https://github.com/Kitware/pan3d/pull/33/commits/28561cb331878bed9f5131df5a01d8307360c41e)